flow/display_list.cc

```diff
@@ -951,17 +951,19 @@ bool DisplayList::Equals(const DisplayList& other) const {
   if (used_ != other.used_ || op_count_ != other.op_count_) {
     return false;
   }
-  if (ptr_ == other.ptr_) {
+  uint8_t* ptr = storage_.get();
+  uint8_t* o_ptr = other.storage_.get();
+  if (ptr == o_ptr) {
```

Comment:
I believe you should be able to compare the unique_ptrs directly.
```
if (storage_ == other.storage_) {
  ...
}
```

Reply:
We need them anyway for the next step.